src/scss/components/_button.scss

```diff
@@ -15,5 +15,21 @@
         border-inline-start-color: transparent;
       }
     }
+
+    // approximate browser default behaviour for focus outline
+    &:focus-visible {
+      --utrecht-button-border-radius: 2px;
+    }
+  }
+}
+
+@include mobile-only {
+  .openforms-theme {
```

Comment:
How come we put the .open-forms theme inside the `mobile-only` instead of adding the `mobile-only` in the rules above?

Reply:
I tend to isolate/group mobile styles from the rest of the styles so that it's a bit more clear which CSS exactly is responsible for what, but it's not exact science :grimacing: